### PR TITLE
Chore/reorder dashboard nav menu

### PR DESCRIPTION
**Pull request by saltcod**

### CODE REVIEW

File: studio/components/layouts/LogsLayout/LogsMenu.utils.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+          ? [
+              {
+                name: 'Storage logs',
+                key: 'storage-logs',
+                url: `/project/${ref}/logs/storage-logs`,
+                items: [],
+              },
+            ]
+          : []),
+        ...(logsRealtime
```
Comment (joshenlim) [inline studio/components/layouts/LogsLayout/LogsMenu.utils.ts:53]: just to verify this is available for self-hosted?

File: studio/components/layouts/LogsLayout/LogsMenu.utils.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
+import { Project } from 'types'
+import { IS_PLATFORM } from 'lib/constants'
+import { ProductMenuGroup } from 'components/ui/ProductMenu/ProductMenu.types'
+import { useFlag } from 'hooks'
+
+export const generateLogsMenu = (project?: Project): ProductMenuGroup[] => {
```
Comment (joshenlim) [inline studio/components/layouts/LogsLayout/LogsMenu.utils.ts:6]: just to check - the entire logs route is hidden behind IS_PLATFORM right? based on NavigationBar.utils?

in that case that i guess we don't have to put IS_PLATFORM here at all then

File: studio/components/layouts/SettingsLayout/SettingsLayout.tsx [hunk 3]
```diff
@@ -18,7 +18,13 @@ const SettingsLayout: FC<Props> = ({ title, children }) => {
   const projectBaseInfo = ui.selectedProjectBaseInfo
 
   const router = useRouter()
-  const page = router.pathname.split('/')[4]
+  let page
```
Comment (joshenlim) [inline studio/components/layouts/SettingsLayout/SettingsLayout.tsx:21]: can we use `const` here? just a best practice to reduce the usage of `let` where we can - immutability

```
const page = router.pathname.includes('billing')
    ? router.pathname.split('/')[5]
    : router.pathname.split('/')[4]
```

File: studio/components/layouts/ProjectLayout/NavigationBar/NavigationBar.utils.tsx [hunk 4]
```diff
@@ -36,6 +36,38 @@ export const generateProductRoutes = (ref?: string, project?: ProjectBase): Rout
...
+  const buildingUrl = `/project/${ref}/building`
+
+  return [
+    {
+      key: 'database',
+      label: 'Database',
+      icon: <IconDatabase size={18} strokeWidth={2} />,
+      link:
+        ref &&
+        (isProjectPaused ? homeUrl : isProjectBuilding ? buildingUrl : `/project/${ref}/editor`),
```
Comment (joshenlim) [inline studio/components/layouts/ProjectLayout/NavigationBar/NavigationBar.utils.tsx:69]: wrong URL

File: studio/components/layouts/AuthLayout/AuthLayout.utils.ts [hunk 5]
```diff
@@ -4,37 +4,26 @@ import { ProductMenuGroup } from 'components/ui/ProductMenu/ProductMenu.types'
...
-              {
-                name: 'Logs',
-                key: 'logs',
-                url: `/project/${ref}/auth/logs`,
-                items: [],
-              },
-            ]
-          : []),
-      ],
+      title: 'Users',
```
Comment (joshenlim) [inline studio/components/layouts/AuthLayout/AuthLayout.utils.ts:7]: this look right? @MildTomato 
image

File: studio/components/interfaces/Auth/Users/Users.tsx [hunk 6]
```diff
@@ -109,7 +109,7 @@ const Users = () => {
         </div>
       </div>
       <section className="users-table-container mt-4 overflow-visible px-6">
-        <div className="section-block--body relative overflow-x-scroll rounded">
+        <div className="section-block--body relative overflow-x-auto rounded">
```
Comment (joshenlim) [inline studio/components/interfaces/Auth/Users/Users.tsx:112]: i added this so that scrollbars will only show up when it starts to overflow

**Review by joshenlim — CHANGES_REQUESTED** [review]
couple of comments - and @MildTomato also mentioned that some UI needs changing so i can review again once that's done

File: studio/components/layouts/LogsLayout/LogsMenu.utils.ts (hunk 2 above)
Comment (saltcod) [reply studio/components/layouts/LogsLayout/LogsMenu.utils.ts:6]: Yes! You're right actually. Good catch.

File: studio/components/layouts/SettingsLayout/SettingsLayout.tsx (hunk 3 above)
Comment (saltcod) [reply studio/components/layouts/SettingsLayout/SettingsLayout.tsx:21]: HAH Wrote that late — what was I thinking there!

File: studio/components/layouts/ProjectLayout/NavigationBar/NavigationBar.utils.tsx (hunk 4 above)
Comment (saltcod) [reply studio/components/layouts/ProjectLayout/NavigationBar/NavigationBar.utils.tsx:69]: hummm isn't ref/editor the right url there?

File: studio/components/interfaces/Auth/Users/Users.tsx (hunk 6 above)
Comment (saltcod) [reply studio/components/interfaces/Auth/Users/Users.tsx:112]: aah nice!

**Review by alaister — APPROVED** [review]
🔥🔥

**Review by joshenlim — APPROVED** [review]

**Pull request merged**